index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PIX TECH未経験から一人で育つエンジニアへ</title>
+  <meta name="description" content="完全現場主義 少数制での、個人個人の堅実なスキルアップの徹底。大手企業を経験したエンジニアが教える、生きた現場のスキル">
+  <link rel="icon" href="./img/favicon.ico">
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+  <link href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="inner">
+      <a href="firstView">
+        <h1 class="logo">
+          <img src="img/logo_sticky.png" alt="PIX TECH">
+        </h1>
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="item"><a href="#">PIX TECHの特徴</a></li>
+          <li class="item"><a href="#">PIX TECHの思い</a></li>
+          <li class="item"><a href="#">カリキュラム</a></li>
+          <li class="item"><a href="#">コースの紹介</a></li>
+          <li class="item"><a href="#">講師の紹介</a></li>
+          <li class="item"><a href="#">よくある質問</a></li>
+          <li class="item"><a href="#" class="btn">説明会へ申し込む</a></li>
+        </ul>
+      </nav>
+      <p class="fas fa-bars fa-lg"></p>
```

Comment:
https://github.com/namitarox/pixtech_yumita/pull/3#discussion_r470390371
このコメントは、
```suggestion
    <p>
      <i class="fas fa-bars fa-lg"></i>
    </p>
```
という意味です。

Reply:
し

> [#3 (comment)](https://github.com/namitarox/pixtech_yumita/pull/3#discussion_r470390371)
> このコメントは、
> 
> という意味です。

修正
4f3e5029357ecf8c7e8928bcfdd243f37a3e5544